widgetry/src/widgets/mod.rs

```diff
@@ -308,6 +308,22 @@ impl Widget {
     pub fn draw_svg_transform(ctx: &EventCtx, filename: &str, rewrite: RewriteColor) -> Widget {
         JustDraw::svg_transform(ctx, filename, rewrite)
     }
+    pub fn draw_svg_with_tooltip<I: Into<String>>(
+        ctx: &EventCtx,
+        filename: I,
+        tooltip: Text,
+    ) -> Widget {
+        let (mut batch, bounds) = crate::svg::load_svg(ctx.prerender, &filename.into());
+        // Preserve the whitespace in the SVG.
+        // TODO Maybe always do this, add a way to autocrop() to remove it if needed.
+        batch.push(Color::INVISIBLE, bounds.get_rectangle());
```

Comment:
👍 I noticed the icons seemed not to be rendered in a uniform size - I assume this fixes that?

Reply:
All the icons are 25x26 or 26x26, except for the removed passenger icon. They looked the same size to me before this PR. `JustDraw` uses the SVG declared dimensions, so this change here matches that. I think your invisible rectangle idea for text generalizes -- if we just added this rectangle to everything returned by `load_svg`, I think we could simplify some code. Might look into that later.